renaud.mk

```diff
@@ -1,4 +1,4 @@
 INSTANCE_ID = renaud
 VARS_FILE = vars_renaud.yaml
-
+PRINT_OUTPUT = /tmp
```

Comment:
OK can we set PRINT_VERSION = NONE in the geoportailv3.mk for now and add to it to main.mk?
So the developers are not bothered with print issues